kasa/smart/modules/energy.py

```diff
@@ -67,13 +69,13 @@ async def get_status(self):
     @raise_if_update_error
     def consumption_this_month(self) -> float | None:
         """Get the emeter value for this month in kWh."""
-        return self.energy.get("month_energy") / 1_000
+        return self.energy.get("month_energy", 0) / 1_000
 
     @property
     @raise_if_update_error
     def consumption_today(self) -> float | None:
         """Get the emeter value for today in kWh."""
-        return self.energy.get("today_energy") / 1_000
+        return self.energy.get("today_energy", 0) / 1_000
```

Comment:
Avoids dividing none if the data is not available for some reason, should be fine.